### DISCUSSION TITLE
Framework :: Aiohttp is not in pypi.org classifiers

### DISCUSSION THREAD

**Discussion by Maillol** [aio-libs/aiohttp#5229] [Ideas] [answered]
Hello,

I don't see aiohttp :: framework in the list of  pypi classifiers.  [Trove classifiers](https://pypi.org/classifiers/)

Have you already try to make a PR on  [pypa/trove-classifiers](https://github.com/pypa/trove-classifiers/pulls) to add it ?

Have a nice day.

**Comment by asvetlov** [comment, marked as answer]
I have an issue: https://github.com/pypa/trove-classifiers/issues/54

No response yet:)

Please feel free to make a PR!

**Reply by Maillol** [reply]
I saw this comment https://github.com/pypa/trove-classifiers/pull/24#issuecomment-617753167

> We understand that "notability" is subjective and we'd appreciate help judging that; responses to the next guideline help.

> As to the other guideline ('immediate use to existing projects'), the best way to demonstrate it is to have 10 (or more) other project maintainers submit comments on the requesting issue about their desire to use the classifier on their already-published project.

If aiohttp side project maintainer can send a comment here:  https://github.com/pypa/trove-classifiers/issues/54

**Reply by webknjaz** [reply]
The PR exists, it's just not yet merged: https://github.com/pypa/trove-classifiers/pull/55.
